### ISSUE TITLE
Configure ESLint to enforce coding style consistency

### ISSUE THREAD

**Issue by sgobotta** [Cambalab/vue-admin#7]
Searching for different options, I think the best choice would be to use [prettier](https://eslint.vuejs.org/user-guide/) because it's less bureaucratic than airbnb. We just want clean and pretty indented/formatted code to enforce a style among contributors.

> The vue-cli installs all the required dependencies to get it running. The bad thing is that it automatically fixes all code with it's default configuration (which is not what we're aiming for: we don't want double quotes, for example).

Prettier needs a custom `.prettierrc.json` file. Looking at the current code, the following rules should not harm the current code so much, plus we intent not to use semicolons and avoid double quotes in plain javascript.

*taken from [prettier configuration](https://prettier.io/docs/en/configuration.html). There are other rules that could be useful.*

```json
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true
}
```

In order to instantiate prettier we'll have to configure an .eslintrc.json file. I found [this one](https://gist.github.com/sgobotta/2e12c8afba42a980482b59c83803775e) from an old project that could do the trick.

Then I've had a little thought about the fix option that runs by default in the **vue-cli-service** at `npm run lint`. I think it would be safer if we disable the auto fix option for that service. We'll be including a [pre-commit hook feature](#158) to prevent pushing dirty code. That way we can use the npm run lint script just as a code state checker.

**Comment by sgobotta** [comment]
https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/eslint-config-airbnb

**Comment by sgobotta** [comment]
Merged in #162
